source/includes/administration/_organizations.md

```diff
@@ -76,7 +84,7 @@ Attributes | &nbsp;
 `billingDay`<br/>*int* | The billing day of the organization. Default value is 1.
 `isBillable`<br/>*boolean* | If the organization is billable this values is true, false otherwise.
 `billingMode`<br/>*enum* | The billing mode of the organization. It could be either "MANUAL" or "CREDIT_CARD". Default value is "MANUAL".
-`tags`<br/>*Array[string]* | Tags associated to the organization.
+`tags`<br/>*Array[string]* | Tags associated to the organization. These include a reseller organization that the tags belong to.
```

Comment:
Does "These include tags belonging to the organization's reseller" make more sense?